curriculum/challenges/english/07-scientific-computing-with-python/learn-numerical-methods-by-finding-the-square-root-of-a-number-using-the-bisection-method/65ef1eac497754cafa12a26c.md

```diff
@@ -0,0 +1,118 @@
+---
+id: 65ef1eac497754cafa12a26c
+title: Step 21
+challengeType: 20
+dashedName: step-21
+---
+
+# --description--
+
+Call the `square_root_bisection` method with the `N` variable as the argument and print the result.
+
+Experiment with larger values.
+
+With this, you have successfully implemented the bisection method to find the square root of a number.
+ 
+# --hints--
+
+Call the `square_root_bisection` method with the `N` variable as the argument and print the result.
+
+```js
+({ 
+    test: () => 
+    {
+        assert.match(code, /square_root_bisection\(N\)/)
```

Comment:
Reminder: change this once the `find_calls` helper is ready.

Reply:
Done.